packages/x-components/src/x-modules/history-queries/components/history-queries-switch.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <BaseSwitch @change="toggle" :value="isEnabled" />
+  <BaseSwitch @change="toggle" :value="isEnabled" aria-label="Queries' history" />
```

Comment:
Is this aria-label necessary? In our base use case we have a label for this switch that tells more about the action.
![image](https://user-images.githubusercontent.com/68222542/201873302-32fbd0ea-cece-4bb4-b470-b6b310af9334.png)

As I'm reading [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby#benefits_and_drawbacks), `aria-labelledby` has highest precedence, so we might be able to keep this `aria-label` by default and add another `aria-labelledby` in the archetype. Is that correct?

Reply:
Hey! As I see it now in the archetype, using a screen reader (I use Screen Reader Chrome extension instead of VoiceOver for sanity purposes xD) will only say "button ticked / button not ticked".
As you said, `aria-labelledby` has highest precedence, but in fact even a second `aria-label `will replace the message on this first one. So I think we're save using this label as 'fallback'.